UI/UI auth login

KoreaMoney opened the pull request:
## What is this PR? :mag:
Auth Login UI는 앞으로 보입니다.
-

## branch

- ui/ui-Auth-login -> develop

## Changes :memo:
로그인 화면만 ui제작되었습니다.
Global style에 anchor를 추가하여 링크 시 링크 색깔이 되는 것을 수정하였습니다.
white와 black은 승인 전으로 적용은 하지 않았습니다. 
일단 ui적용 시 #컬러로 적용했으며 승인될 경우 추후 기능 구현 시 white 또는 black으로 수정하겠습니다. 

-
#8 
by @dollorofkorea

KoreaMoney committed:
feat: UI - Auth-Login으로 UI만 제작되었습니다.

frankkim92 reviewed (dismissed):
고생하셨습니다.😀

Jeremy-Kr commented on src/components/auth/Login.tsx:
```diff
@@ -0,0 +1,159 @@
+  margin-top: 200px;
+  padding: 20px;
+`;
+const LoginText = styled.text`
+  display: flex;
+  justify-content: center;
+  font-size: ${(props) => props.theme.fontSize.h6};
+`;
```
text라는 태그는 없는걸로 알고있습니다! H6을 상속해서 사용해 보시는건 어떨까요?

Jeremy-Kr replied:
아래에 나머지 text 태그들도 마찬가지 의견입니다.

KoreaMoney replied:
네.오늘 수정해서 다시 커밋하겠습니다.

Jeremy-Kr reviewed (changes requested):
text 태그... 말고 다른 태그를 상속해서 사용해봅시다!

KoreaMoney commented:
네 그럼 오늘 수정해서 다시 커밋하겠습니다

KoreaMoney committed:
feat : branch잘못 지정해서 재 지정후 진행
login 색상 : white와 black으로 색상 props사용
login text :  textComponents이용 ( text태그 사용 안함 )

KoreaMoney commented:
## What is this PR? :mag:
Auth Login UI는 앞으로 보입니다. 
-

## branch

- ui/ui-Auth-login -> develop

## Changes :memo:
textComponents 사용으로 통일 (기존 text태그 삭제)
최신  dev merge
색상 white와 black 으로 변경
-
#8 
by @dollorofkorea

Jeremy-Kr reviewed (approved):
LGTM

harenohee reviewed (approved):
확인했습니다. 🥹

harenohee reviewed (approved):
H6 태그 적용 확인했습니당
